core/interfaces/rebase.py

```diff
@@ -212,8 +212,9 @@ def render_conflicts_bindings(self):
     def base_ref(self):
         base_ref = self.view.settings().get("git_savvy.rebase.base_ref")
         if not base_ref:
-            base_ref = "master"
-            self.view.settings().set("git_savvy.revase.base_ref", "master")
```

Comment:
There was a typo here that I corrected (revase => rebase), but I don't think was actually causing any issues.